build.cmd

```diff
@@ -67,7 +67,7 @@ set "__IntermediatesDir=%__RootBinDir%\obj\%__BuildOS%.%__BuildArch%.%__BuildTyp
 set "__PackagesBinDir=%__BinDir%\.nuget"
 set "__TestBinDir=%__RootBinDir%\tests\%__BuildOS%.%__BuildArch%.%__BuildType%"
 set "__TestIntermediatesDir=%__RootBinDir%\tests\obj\%__BuildOS%.%__BuildArch%.%__BuildType%"
-set "__GeneratedIntermediatesDir=%__IntermediatesDir%\Generated"
+set "__GeneratedIntermediatesDir=%__IntermediatesDir%\Generated_present" ::use this variable to locate dynamically generated files, the actual location though will be different
```

Comment:
NIT: It might be better to call the dir name `Generated_DiffRight` because Present is used in the batch variable name below without the _present suffix.